rss_to_json.ipynb

```diff
@@ -38,7 +38,7 @@
    "outputs": [],
    "source": [
     "# Define regex for parsing description field (includes date, time, description, number of attendees, price, etc)\n",
-    "expression = re.compile('.*<p><p>(?P<description>.*)<\\/p> <\\/p> <p>(?P<location>.*)<\\/p> <p>(?P<date>.*)<\\/p> <p>(?P<attendees>.*)<\\/p> <p>Price: (?P<price>[0-9]+).*<\\/p> <p>(?P<url>.*)<\\/p>.*')\n",
+    "expression = re.compile('.*<p><p>(?P<description>.*)<\\/p> <\\/p> <p>(?P<location>.*?)<\\/p> <p>(?P<date>.*?)<\\/p> <p>(?P<attendees>.*?)<\\/p>( <p>Price: (?P<price>[0-9]+).*?<\\/p>)? <p>(?P<url>.*)<\\/p> ]]>.*')\n",
```

Comment:
The brackets at the end are not in the real strings. Can you remove those.

e.g. `.*<p><p>(?P<description>.*)<\/p> <\/p> <p>(?P<location>.*?)<\/p> <p>(?P<date>.*?)<\/p> <p>(?P<attendees>.*?)<\/p>( <p>Price: (?P<price>[0-9]+).*?<\/p>)? <p>(?P<url>.*)<\/p>.*`

Otherwise this works perfectly!

Reply:
https://regex101.com/r/HLCFll/4